tracer_test.go

```diff
@@ -53,7 +53,9 @@ func (s *tracerSuite) SetupTest() {
 		NewConstSampler(true),
 		NewNullReporter(),
 		TracerOptions.Metrics(metrics),
-		TracerOptions.HostIPv4(IP))
+		TracerOptions.HostIPv4(IP),
+		TracerOptions.ZipkinSharedRPCSpan(true),
```

Comment:
so it's true by default in this test suite, and it's set to true in the new dedicated test, but where do we test the case when it is NOT true?